scout/server/blueprints/cases/controllers.py

```diff
@@ -189,8 +189,7 @@ def case(store, institute_obj, case_obj):
                                    collab_obj in o_collaborators]
 
     collab_ids = [(collab['_id'], collab['display_name']) for collab in store.institutes() if
-                  (collab['_id'] not in ('cust000', institute_obj['_id'])) and
-                  (collab['_id'] not in case_obj['collaborators'])]
+        collab['_id'] in institute_obj.get('collaborators')
```

Comment:
:+1:

Reply:
You are fast - but you shouldn't really be working on this.. I just found the right function a minute ago, but was checking what o_collaborators did. Turns out just for the revoke I, so this is perfect!

Reply:
Last contributions, and probanly doesn't work, shoulf be fixed

Reply:
Ah, right, tried it now. But, right place. I'm on it!